package.json

```diff
@@ -39,7 +39,7 @@
     "mocha": "^8.1.3"
   },
   "engines": {
-    "node": "^12.20.0 || ^14.13.1 || >=16.0.0"
+    "node": "^12.17.0 || ^14.13.1 || >=16.0.0"
```

Comment:
It should be `^12.20.0 || ^14.13.1 || >=16.0.0`. Because we are using Meow, it requires `^12.20.0 || ^14.13.1 || >=16.0.0`.

Reply:
Ah damn, of course :sweat_smile: thanks for catching that mistake!